src/sentry/services/hybrid_cloud/integration/impl.py

```diff
@@ -128,7 +128,7 @@ def get_organization_integrations(
         status: int | None = None,
         providers: List[str] | None = None,
         has_grace_period: bool | None = None,
-        limit: int | None = 5,
+        limit: int | None = None,
```

Comment:
It was silly in the first place to arbitrarily set a limit like this without enforcing it as a required param. Having no limit makes it function as intended, while allowing areas that require better performance to still specify the maximum results they expect to receive.